m2m/tag_mapper.py

```diff
@@ -24,6 +25,23 @@ def to_markdown(self):
             return self.markdown_unordered_list()
         elif self.tag.name == 'ol':
             return self.markdown_ordered_list()
+        elif self.tag.name == 'div':
+            return self.parse_div(self.tag)
+
+    def parse_div(self, div_tag):
+        for child in div_tag:
+            if child.name == 'div':
+                self.s += self.parse_div(child) if self.parse_div(child) else ''
+            elif child.name == 'a' and child.has_attr('href'):
+                link_text = reduce(lambda result_text, current_text: self.parse_text(
+                    result_text, current_text), child, "")
+                self.s += f"[{link_text}]({child['href']})"
+            self.s += temp
```

Comment:
I think we don't have the temp `variable` in this method. Is it broken, isn't it?